packages/trader/src/Modules/Trading/Components/Form/TradeParams/Accumulator/accumulator.tsx

```diff
@@ -1,23 +1,30 @@
-import { PropTypes as MobxPropTypes } from 'mobx-react';
-import PropTypes from 'prop-types';
 import React from 'react';
 import { localize } from '@deriv/translations';
-import NumberSelector from 'App/Components/Form/number-selector.jsx';
-import Fieldset from 'App/Components/Form/fieldset.jsx';
-import { connect } from 'Stores/connect';
+import NumberSelector from '../../../../../../App/Components/Form/number-selector.jsx';
+import Fieldset from '../../../../../../App/Components/Form/fieldset.jsx';
+import { connect } from '../../../../../../Stores/connect';
```

Comment:
same here: please remove connect and change the approach to using useStore

Reply:
answered above lol

Reply:
@henry-deriv please remove all unnecessary multiple ../